### PR TITLE
adding join us page

### PR DESCRIPTION
**Pull request by mmilenkovic**

### CODE REVIEW

**Comment by adrifdez** [comment]
@mmilenkovic could you check this pr as well that I can't assignee you because you did some changes too?

File: src/components/join-us/join-us.component.jsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+              </div>
+              <p className={classes.paragraphMargin}>
+                It’s very exciting times for us here at Hermez Project.
+                Following our successful launch and expansion plans, we are
+                growing massively.
+              </p>
+              <p className={classes.paragraphMargin}>
+                You can see our list of vacancies below, but we’re always
+                looking for top talent, blockchain enthusiasts and if you think
+                you have what it takes to help us achieve our mission, please do
```
Comment (AlbertoElias) [inline src/components/join-us/join-us.component.jsx]: The phrase have what it takes seems quite aggressive and might put off non-white male folks

Comment (adrifdez) [reply src/components/join-us/join-us.component.jsx]: Could you rephrase it?

Comment (AlbertoElias) [reply src/components/join-us/join-us.component.jsx]: and if this sounds interesting to you, please do.... but you'd need to check with ofra

Comment (adrifdez) [reply src/components/join-us/join-us.component.jsx]: I've approval for this, What do you think?
_"You can see our list of vacancies below. In addition, we're always looking for top talent, blockchain enthusiasts and if this sounds interesting to you, please do get in touch – we always try to work out how we can fit great people into our team."_

Comment (AlbertoElias) [reply src/components/join-us/join-us.component.jsx]: That is so much better, I like it!

**Review by AlbertoElias — APPROVED** [review]

**Comment by mmilenkovic** [comment]
> @mmilenkovic could you check this pr as well that I can't assignee you because you did some changes too?

Yes, sorry I missed this message.

**Review by mmilenkovic — COMMENTED** [review]
All good!

**Comment by mmilenkovic** [comment]
@adrifdez Merge it, and let's see it in action :)

**Review by AlbertoElias — APPROVED** [review]
